From 8c69843abc2e7246d175f75df7ddafed4f2accda Mon Sep 17 00:00:00 2001
From: Krzysztof Kozlowski <krzysztof.kozlowski@proceq.com>
Date: Wed, 6 Jun 2018 12:34:02 +0200
Subject: [PATCH] Revert "fix #714 (system width detection)"

This reverts commit cb3c2999b586aaef7e5bd5ba60f1e3631ec756c1.  Needed to
build with musl.

Signed-off-by: Krzysztof Kozlowski <krzk@kernel.org>
---
 src/core/abi.cc | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/core/abi.cc b/src/core/abi.cc
index adff7b55acfa..76e508238c18 100644
--- a/src/core/abi.cc
+++ b/src/core/abi.cc
@@ -20,9 +20,7 @@ __ID("@(#) $Id: mem.cc 1352 2006-05-27 23:54:13Z ezix $");
 bool scan_abi(hwNode & system)
 {
   // are we compiled as 32- or 64-bit process ?
-  long sc = sysconf(LONG_BIT);
-  if(sc==-1) sc = sysconf(_SC_LONG_BIT);
-  if(sc!=-1) system.setWidth(sc);
+  system.setWidth(sysconf(LONG_BIT));
 
   pushd(PROC_SYS);
 
